lib/multiplex.dart

```diff
@@ -29,6 +29,10 @@ class Server {
 
   Server(this.port, this.chrome, {this.modelDom}) {
     _server = io.serve(_handler, InternetAddress.ANY_IP_V4, port);
+      
+    // Allows cross origin.
+    _server.then((httpServer) =>
```

Comment:
Do we think this is something all clients will want? If no, we should probably pass in as a flag (`bool stripFrameOptions: false`), w/ some docs to describe what it does, or perhaps pass in a closure that has an opportunity to act on the `HttpServer`.